substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/pecoff/cv/CVTypeSectionImpl.java

```diff
@@ -63,12 +83,14 @@ public String getSectionName() {
     public void createContent(DebugContext debugContext) {
         int pos = 0;
         enableLog(debugContext);
+        builder.setDebugContext(debugContext);
         log(debugContext, "CVTypeSectionImpl.createContent() adding records");
         addRecords();
         log(debugContext, "CVTypeSectionImpl.createContent() start");
         pos = CVUtil.putInt(CV_SIGNATURE_C13, null, pos);
         for (CVTypeRecord record : typeMap.values()) {
             pos = record.computeFullSize(pos);
+            pos = CVUtil.pad4(null, pos);
```

Comment:
I was getting a lot of messages from the codeview dump about misaligned records; this was part of an effort to fix it.  Turns out the linker will emit multiple class definitions and screw up alignment anyways, so probably this can be removed.

Reply:
removed.